python/lsst/analysis/drp/colorColorPlot.py

```diff
@@ -152,8 +166,12 @@ def colorColorPlot(self, catPlot, plotInfo):
                        self.config.connections.plotName, self.config.axisLabels["x"],
                        self.config.axisLabels["y"])))
 
-        fig = plt.figure()
-        ax = fig.add_axes([0.12, 0.11, 0.65, 0.75])
+        # Define a new colormap
+        newBlues = mkColormap(["paleturquoise", "midnightblue"])
+        newReds = mkColormap(["lemonchiffon", "firebrick"])
+
+        fig = plt.figure(dpi=200)
```

Comment:
I think this needs to be a higher `dpi`. On my screen, 200 DPI still looks a bit pixelated. Would recommend increasing to 300 DPI; 300 DPI is the standard print resolution for high resolution output